indexer/src/main/java/org/jboss/windup/rules/apps/java/archives/SortingLineWriterArtifactVisitor.java

```diff
@@ -0,0 +1,85 @@
+package org.jboss.windup.rules.apps.java.archives;
+
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Logger;
+import org.apache.commons.collections4.list.TreeList;
+import org.apache.maven.index.ArtifactInfo;
+
+/**
+ *
+ *  @author <a href="http://ondra.zizka.cz/">Ondrej Zizka, zizka@seznam.cz</a>
+ */
+public class SortingLineWriterArtifactVisitor implements RepositoryIndexManager.ArtifactVisitor<Object>
```

Comment:
This one creates the text file with metadata. I'd prefer to keep that class as it can come in handy.